### DISCUSSION TITLE
Does SketchyBar work with Bartender?

### DISCUSSION THREAD

**Discussion by szinn** [FelixKratz/SketchyBar#465] [Q&A]
I currently use Bartender to manage the menubar icons on the right side of the menu and just found sketchybar. I still want bartender functionality but am intrigued with managing stuff on the right side.

**Comment by Fau818** [comment]
I guess you no longer need the Bartender if you use SketchyBar.

The functions provided by Bartender can be implemented as a click event in SketchyBar.
